cli/syslog.py

```diff
@@ -0,0 +1,23 @@
+"""Syslog constants."""
```

Comment:
Good point, and thanks for the prompt to have another log at the syslog facilities. Initially I had added in all of the facilities that Python's `logging.handlers.SysLogHandler` had support for, to retain a certain amount of flexibility (and because I also don't know a ton about syslog, other than what I've read since noticing that the CEF specification puts emphasis on using it when possible).

Having looked again, I don't think it's appropriate for `auditmatica` to write to many of those facilities, so I have kept the default of `USER` and restricted the `--syslog-facility` options to only `LOCAL0-LOCAL7`, which are facilities whose purpose is left open for applications to use. This is closer to what other applications do, e.g. [Postgres](https://postgresqlco.nf/doc/en/param/syslog_facility/).

Two sources I found helpful on the subject.
- https://en.wikipedia.org/wiki/Syslog#Facility
- https://unix.stackexchange.com/questions/90842/what-is-the-local6-and-all-other-local-facilities-in-syslog

I'll update the `cli/syslog.py` module docstring and README accordingly.